features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/developer/DeveloperSettingsState.kt

```diff
@@ -16,10 +16,12 @@
 
 package io.element.android.features.preferences.impl.developer
 
+import io.element.android.libraries.architecture.Async
 import io.element.android.libraries.featureflag.ui.model.FeatureUiModel
 import kotlinx.collections.immutable.ImmutableList
 
-data class DeveloperSettingsState(
+data class DeveloperSettingsState constructor(
```

Comment:
TIL: You can still call "Find Usages" on the default primary constructor just by command clicking or right clicking on the opening parenthesis (i.e. click on `(` instead of `constructor`).